### ISSUE TITLE
Cannot install Nuget.Common in csproj

### ISSUE THREAD

**Issue by natemcmaster** [NuGet/Home#2635]
This does not work on a csproj (ironically)

```
Install-Package Nuget.Common -Pre -Source https://www.myget.org/F/aspnetcirelease/
```

```
Install-Package : Failed to add reference to 'System.Runtime'. Please make sure that it is in the Global Assembly Cache.
At line:1 char:1
+ Install-Package Microsoft.EntityFrameworkCore.Tools -Pre -Source C:\U ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
```

https://github.com/NuGet/NuGet.Client/blob/2e9de8ec7fe9fd5de4d2b608eb5cedb92d046458/src/NuGet.Core/NuGet.Common/project.json#L20-L24
If I'm not mistaken, shouldn't these be `"type": "build"`?

**Comment by rrelyea** [comment]
these should be removed if possible...other wise changing all 5 of those to type : build would be right.
/cc @emgarten

**Comment by natemcmaster** [comment]
Can we get this in ASAP? This is blocking Entity Framework's design-time tools for RC2. cref https://github.com/aspnet/EntityFramework/pull/5142

**Comment by natemcmaster** [comment]
Found more: https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.RuntimeModel/project.json#L30. NuGet.RuntimeModel and others also do not install

https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Packaging/project.json#L37-L40

https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Packaging.Core/project.json#L28-L30

**Comment by jainaashish** [comment]
https://github.com/NuGet/NuGet.Client/pull/524

**Comment by natemcmaster** [comment]
Thanks @jainaashish Is there a public feed for consuming these latest builds?

**Comment by zhili1208** [comment]
Verified fixed with 3.5.0-beta (build 3.5.0.1249)
